src/css/how_its_made.css

```diff
@@ -0,0 +1,196 @@
+
+@media screen and (min-width: 378px) and (max-width: 1140px) {
+    body {
+        background-color: #FAFAFA ;
+    }
+
+    .container {
+        margin: 0 auto;
+        padding: 0 20px;
+    }
+
+    .how-its-made-container {
+        display: block;
+    }
+
+    .how-its-made-heading {
+        color: var(--black);
+        text-align: center;
+        font-family: var(--font);
+        font-size: 20px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 1.2;
+        letter-spacing: -0.6px;
+        text-transform: uppercase;
+        margin-bottom: 32px;
+    }
+    
+    .how-its-made-heading-orange {
+        color: var(--orange);
+    }
+
+    .flex-container {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+    }
+
+    .how-its-made-img {
+        max-width: 100%;
+        height: 500px;
+        object-fit: cover;
+        border-radius: 15px;
+        margin-bottom: 20px;
+    }
+
+    .background-div {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        flex-direction: column;
+        width: 100%;
+        background: white;
+        border-radius: 15px;
+    }
+    .background-container {
+        padding: 20px 14px;
+    }
+    .try-our-chocolates {
+        color: var(--black);
+        font-family: var(--font);
+        font-size: 18px;
+        font-style: normal;
+        font-weight: 600;
+        line-height: 1.33;
+        max-width: 307px;
+    }
+    
+    .how-its-made-list {
+        padding-left: 14px;
+    }
+
+    .how-its-made-list-item {
```

Comment:
ustaw sobie tutaj tez list type i kropke

Reply:
jak jest w figmie, bo typ listy w globals jest na none, wiec dla tej klasy nadpisz to.